src/Umbraco.Web/Security/Providers/UmbracoMembershipProvider.cs

```diff
@@ -548,6 +547,7 @@ public override bool ValidateUser(string username, string password)
             }
 
             var authenticated = CheckPassword(password, member.RawPasswordValue);
+            var backofficeUserManager = GetBackofficeUserManager();
```

Comment:
This method isn't used to log in users anymore so we don't need to have this logging in here. This is also one of the scenarios where this is shared between members and users and now we have custom checks for user specific things in here.